doc/controls/AutoLayoutControl.md

```diff
@@ -22,30 +25,33 @@ xmlns:utu="using:Uno.Toolkit.UI"
 </utu:AutoLayout>
 ```
 
-### Inheritance 
+### Inheritance
+
 Object &#8594; DependencyObject &#8594; UIElement &#8594; FrameworkElement &#8594; Control
 
 ### Constructors
+
 | Constructor     | Description                                              |
 |-----------------|----------------------------------------------------------|
 | AutoLayout() | Initializes a new instance of the `AutoLayout` class. |
 
 ### Properties
+
 Property|Type|Description
 -|-|-
-Orientation | Orientation | Defines constants that specify the different orientations that a control or layout can have. 
-Spacing | double | Gets or sets the uniform distance (in pixels) between grid. Each grid of the AutoLayout will have the spacing set by this value depending the AautoLayout orientation.
-Justify | AutoLayoutJustify | Either Stack, space between children behave like a `StackPanel`, or SpaceBetween, between each children a equal space is added assuring that all the `AutoLayout` space is occupy. Note : if a child has his `PrimaryAlignment` set to Stretch, it will default to Stack.
-Children |  AutoLayoutChildren | List of FrameworkElement that are the AutoLayout children.
-PrimaryAxisAlignment | AutoLayoutAlignment | Indicates where the content be displayed on the axis of the AutoLayout orientation.
-IsReverseZIndex | bool | Reverse the Z-index of all the children.
-Padding | Thickness | /!\ Padding for `AutoLayout` behave the same as it does within the Figma Plugin: The anchor points determine which sides of the Padding will taken into consideration. For example, items that are aligned to the Right and Top positions will only take the `Tickness.Right` and `Thickness.Top` values of `Padding` into consideration.
+Orientation | `Orientation` | Gets or sets the dimension by which the items are stacked
+Spacing | `double` | Gets or sets a uniform distance (in pixels) between stacked items. It is applied in the direction of the `AutoLayout`'s Orientation
+Justify | `AutoLayoutJustify` | Gets or sets the value to determine how items are justified within the container. Options are `Stack` or `SpaceBetween`. Note: if a child has its `PrimaryAlignment` set to `Stretch`, it will default to `Stack`
+PrimaryAxisAlignment | `AutoLayoutAlignment` | Gets or sets the alignment characteristics that are applied to the content, based on the value of the `Orientation` property. Options are `Start`, `Center`, `End`, and `Stretch`. The default is `Start`
```

Comment:
Stretch is not use here. It is set in the primary alignement of the children

Reply:
https://github.com/unoplatform/uno.toolkit.ui/blob/24c3acf60b96f8cd84415deb528ea80cf3f3e299/src/Uno.Toolkit.UI/Controls/AutoLayout/AutoLayoutAlignment.cs#L4

The enum has a Stretch value. Is it just ignored if its used for PrimaryAxisAlignment?

Reply:
the enum AutoLayoutAlignment is use for other properties that use stretch (like CounterAlignement)

Reply:
https://github.com/unoplatform/uno.toolkit.ui/blob/775b4660781021d847da76f32e1b077801153180/src/Uno.Toolkit.UI/Controls/AutoLayout/AutoLayout.Properties.cs#L67-L78

It's also used for this property. Should I just say that it's ignored for this property specifically?

Reply:
@Robert-Louis what should i change this to?

Reply:
poke @Robert-Louis